test/k8s-integration/main.go

```diff
@@ -352,6 +355,9 @@ func generateGCETestSkip(clusterVersion string) string {
 	if v.LessThan(apimachineryversion.MustParseSemantic("1.16.0")) {
 		skipString = skipString + "|volumeMode\\sshould\\snot\\smount\\s/\\smap\\sunused\\svolumes\\sin\\sa\\spod"
 	}
+	if platform == "windows" {
+		skipString = skipString + "|\\[Slow\\]|\\[LinuxOnly\\]|\\[Feature:.+\\][Ephemeral]"
```

Comment:
What slow, and feature tests don't work on windows? Also ephemeral can probably be removed, there shouldn't be any external storage tests tagged with ephemeral

Reply:
en, I forgot why I added slow.  I will remove it.